Bricknode.Soap.Sdk/Services/BfsOrderService.cs

```diff
@@ -368,6 +387,54 @@ public async Task<CreateAutoGiroDepositOrderResponse> CreateAutoGiroDepositOrder
             return response;
         }
 
+        /// <summary>
+        /// https://bricknode.atlassian.net/wiki/spaces/API/pages/3744595969/CancelAutoGiroOrders
+        /// </summary>
+        public async Task<CancelAutoGiroOrderResponse> CancelAutoGiroOrdersAsync(Guid[] orderIds, string? bfsApiClientName = null)
+        {
+            var request = await GetRequestAsync<CancelAutoGiroOrderRequest>(bfsApiClientName);
+
+            var listOfCancelTradeOrder = new List<CancelAutoGiroOrder>();
+
+            foreach (var tradeOrderId in orderIds)
+            {
+                listOfCancelTradeOrder.Add(new CancelAutoGiroOrder
+                {
+                    OrderId = tradeOrderId
```

Comment:
This method does not map the cancelComment from the linked documentation, is that on purpose?

Reply:
I wanted to be consistent with CancelTradeOrders, where is one overload with just Ids and one with entire model.